util/tracecpu/collector.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracecpu
+
+// TopSQLCollector uses to collect SQL stats.
+// TODO: add a collector to collect and store the SQL stats.
+type TopSQLCollector interface {
+	Collect(ts int64, stats []SQLStats)
+	RegisterSQL(sqlDigest, normalizedSQL string)
+	RegisterPlan(planDigest string, normalizedPlan string)
+}
+
+// SQLStats indicate the SQL stats.
+type SQLStats struct {
```

Comment:
`SQLStats` is too general
What about renaming it to `TopSQLRecord`, which means this is a single record about TopSQL to be collected?

Reply:
Great, reasonable.